.azure-pipelines/gpu-tests.yml

```diff
@@ -32,7 +32,7 @@ jobs:
       image: "pytorchlightning/pytorch_lightning:base-cuda-py3.7-torch1.8"
       # default shm size is 64m. Increase it to avoid:
       # 'Error while creating shared memory: unhandled system error, NCCL version 2.7.8'
-      options: "--runtime=nvidia -e NVIDIA_VISIBLE_DEVICES=all --shm-size=256m"
+      options: "--runtime=nvidia -e NVIDIA_VISIBLE_DEVICES=all --shm-size=512m"
```

Comment:
with setting `--ipc=host` we could allow containers to use the hosts shared memory instead of manually controlling its size. Do we want that or are we fine with manually increasing this potentially multiple times here?

Reply:
@carmocca Good point. Might be worth to use `--ipc=host`